docs/how-to-guides/block-tutorial/nested-blocks-inner-blocks.md

```diff
@@ -63,7 +63,12 @@ Specifying this prop does not affect the layout of the inner blocks, but results
 By default `InnerBlocks` opens a list of permitted blocks via `allowedBlocks` when the block appender is clicked. You can modify the default block and its attributes that are inserted when the initial block appender is clicked by using the `defaultBlock` property. For example:
 
 ```js
-<InnerBlocks defaultBlock={['core/paragraph', {placeholder: "Lorem ipsum..."}]} directInsert />
+<InnerBlocks
+	defaultBlock={
+		{ name: 'core/paragraph', attributes: { content: 'Lorem ipsum...' } }
+	}
+	directInsert={true}
```

Comment:
```suggestion
	directInsert
```

Let's revert this to truthy shorthand, a recommended way of passing boolean values. Core has an ESLint rule for it (#59557), though I'm not sure if it's applied to code in the docs.